packages/@aws-cdk/aws-efs/lib/efs-file-system.ts

```diff
@@ -0,0 +1,260 @@
+import * as ec2 from '@aws-cdk/aws-ec2';
+import * as kms from '@aws-cdk/aws-kms';
+import {Construct, Resource, Tag} from "@aws-cdk/core";
+import {CfnFileSystem, CfnMountTarget} from "./efs.generated";
+import {FileSystemProps, IFileSystem} from "./file-system";
+
+/**
+ * EFS Lifecycle Policy, if a file is not accessed for given days, it will move to EFS Infrequent Access.
+ *
+ * @see http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-efs-filesystem.html#cfn-efs-filesystem-performancemode
+ */
+export enum EfsLifecyclePolicyProperty {
+    /**
+     * After 7 days of inaccessibility.
```

Comment:
Not "inaccessibility", but "not being accessed"

Reply:
Fixed.